charts/oidc-proxy/templates/configmap.yaml

```diff
@@ -119,4 +120,12 @@ data:
 
             ${FILTER3}
         </Location>
+
+        # Bigquery redirect logic.  Note: we are not authorizing the request since that is handled by the bigquery API
+        # This is here to work around CORS rules that are enforced by the BigQuery APIs.  This makes it so that ours are
+        # used instead.
+        <Location /bigquery>
+            ProxyPass https://bigquery.googleapis.com/bigquery
+            ProxyPassReverse https://bigquery.googleapis.com/bigquery
```

Comment:
With my PR, these should actually be passed through to the UI layer. That way, there's absolutely no difference between the local and deployed environments.

Reply:
Basically, keeping the functionality DRY and letting only one place to the actual redirect. This just needs to let the `/bigquery` request to the proxy be routed to the frontend, which actually proxies the request to BigQuery.

Reply:
I don't think it will work that way.  I think your change only affects local dev environments (e.g. when it's hosting with node). It doesn't fix the case where it's being hosted by nginx (which is what the deployed instances do)

Reply:
You know, you're totally right. I got my layers mixed up 😅

Reply:
Hahaha totally understandable...it's super confusing.  I like the change you made...I wasn't looking forward to figuring out how to run the proxy locally